natss/pkg/dispatcher/dispatcher_test.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dispatcher
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/cloudevents/sdk-go/pkg/cloudevents/types"
+
+	"github.com/google/go-cmp/cmp"
+
+	cloudevents "github.com/cloudevents/sdk-go"
+
+	stan "github.com/nats-io/go-nats-streaming"
+	"github.com/nats-io/go-nats-streaming/pb"
+)
+
+func TestSerialize(t *testing.T) {
```

Comment:
Can you add `TestSerialize` & `TestToEvent` cases with minimal event (only required attributes) and with/without data?

Reply:
Done!